pmm-app/src/pmm-check/components/FailedChecksTab/FailedChecksTab.utils.ts

```diff
@@ -0,0 +1,26 @@
+import { logger } from '@percona/platform-core';
+import { SHOW_SILENCED_VALUE_KEY, SHOW_SILENCED_DEFAULT } from './FailedChecksTab.constants';
+
+export const loadShowSilencedValue = (): boolean => {
```

Comment:
Is it really a requirement to store this value?

Reply:
yes, it was discussed with @atymchuk